src/pages/host/HostVanDetail.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+import { Link, Outlet, useParams } from "react-router-dom";
+import { HostVanLayout } from "../../components/layouts";
+
+const HostVanDetail = () => {
+  const { id } = useParams();
+  const [currentVan, setCurrentVan] = useState({});
+  useEffect(() => {
+    fetch(`/api/host/vans/${id}`)
+      .then((res) => res.json())
+      .then((data) => setCurrentVan(data.vans));
```

Comment:
You can also add a way to catch errors, because if an error occurs the app will not do anything.
```suggestion
    fetch(`/api/host/vans/${id}`)
      .then((res) => res.json())
      .then((data) => setCurrentVan(data.vans))
      .catch((error) => setError(error))
```